librz/include/rz_debug.h

```diff
@@ -387,7 +387,11 @@ typedef struct rz_debug_plugin_t {
 	int (*step)(RzDebug *dbg);
 	int (*step_over)(RzDebug *dbg);
 	int (*cont)(RzDebug *dbg, int pid, int tid, int sig);
+#if __WINDOWS__
+	RzDebugReasonType (*wait)(RzDebug *dbg, int pid);
```

Comment:
Why just not use it for everyone? This looks wrong, to be honest

Reply:
Hmm... Agreed. So shall I modify the function signatures for the rest of the functions as well?

Reply:
yes

Reply:
Many of the other functions, like `rz_debug_qnx_wait`, `rz_debug_bochs_wait` return integers as their return value. Directly changing the signature will probably won't work.

Reply:
Why? It will solve the problem of the type mismatch.

Reply:
It will require changing the function internals, since right now many of those return true or false or negative integers. We need to map those return values correctly to `RzDebugReasonType`. Not a big task, but could maybe introduce regressions. Anyways, I'll push a commit doing the forementioned. You can review it, then.